masappcli/masappcli.py

```diff
@@ -314,49 +337,40 @@ def store_scan_info_from_package_name_origin(self, package_name_origin):
         raise ValueError("Application {package_name_origin} not found".format(package_name_origin=package_name_origin))
 
 
-    def store_scan_info_from_package_name(self, app_path):
+    def store_scan_info_from_app_hashPath(self, app_hasPath):
         """
 
-        :param app_path: The absolute path to the application which the user wants to upload.
-        :type  app_path: String
+        :param app_hasPath: The sha1 of the application to analyse
+        :type  app_hasPath: String
         :return:         It obtains all the scans from the user with the given credentials and look for the
-                         scan whose packageName is contained in the name of the file (app_path).
-                         This field is not always generated by mASAPP, so if it is not in the API response it makes the
-                         same comparison with the packageNameOrigin.
-
-                            * If it is found, it returns True and store the scanId and scanDate in scan_info.
-                            * If it is not found, it throw an error and a list of the different scans of the user for\
-                            adding it manually.
+                         scan whose hashPath is equal to the given one.
 
         """
+        time.sleep(10)  # The upload and analysis process is asynchronous so it will wait for the analysis
 
-        user_scans = self.auth_user.get_auth_scans(self.scan_info["wg"])
-        self._check_not_api_error(user_scans)
-        for scan in user_scans.data['data']['scans']:
-            if 'packageName' in scan.keys():
-                if scan['packageName'] in app_path or scan['packageNameOrigin'] in app_path:
-                    self.scan_info['scanId'] = scan['scanId']
-                    self.scan_info['scanDate'] = scan['lastScanDate']
-                    self.scan_info['packageNameOrigin'] = scan['packageNameOrigin']
-                    return True
-            else:
-                if scan['packageNameOrigin'] in app_path:
-                    self.scan_info['scanId'] = scan['scanId']
-                    self.scan_info['scanDate'] = scan['lastScanDate']
-                    self.scan_info['packageNameOrigin'] = scan['packageNameOrigin']
-                    return True
+        if self.scan_info["wg"] == None:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/app/alopezna5/mASAPP_CI/pullRequest?prid=4655639)